crates/eip4844/src/env_settings.rs

```diff
@@ -0,0 +1,75 @@
+use super::trusted_setup_points::{G1_POINTS, G2_POINTS};
+use alloc::sync::Arc;
+use c_kzg::KzgSettings;
+use core::hash::{Hash, Hasher};
+
+/// KZG settings.
+#[derive(Clone, Debug, Default, Eq)]
+pub enum EnvKzgSettings {
```

Comment:
Default kzg settup can be found in `c-kzg` lib behind `ethereum_kzg_settings` features, this would slim down this PR by a lot. 

It is a function `c_kzg::ethereum_kzg_settings()`

Reply:
you meant that Default option can be remove, correct?

Reply:
@mattsse removing functionality from here is breaking change. shall I remove anything from env_settings?